### PR TITLE
[MI-2088]: Review fixes of #872

### PR DESCRIPTION
**Pull request by Nityanand13**
[ Review fixes of #872](https://github.com/mattermost/mattermost-plugin-jira/pull/872)

### CODE REVIEW

**Review by Kshitij-Katiyar — APPROVED** [review]

File: server/settings.go [hunk 1]
```diff
@@ -14,14 +14,15 @@ const (
 
 	errStoreNewSettings = "Could not store new settings. Please contact your system administrator. Error: %v"
 	errConnectToJira    = "Your account is not connected to Jira. Please type `/jira connect`. %v"
-	subCommandAssignee  = "assignee"
-	subCommandMention   = "mention"
-	subCommandReporter  = "reporter"
-	subCommandWatching  = "watching"
+
+	notificationRoleAssignee = "assignee"
+	notificationRoleMention  = "mention"
```
Comment (Kshitij-Katiyar) [inline server/settings.go:19]: you have defined same strings as constant twice with different variable give it a generic name initialise it only once

File: server/webhook_worker.go [hunk 2]
```diff
@@ -39,10 +39,12 @@ func (ww webhookWorker) process(msg *webhookMessage) (err error) {
 	if err != nil {
 		return err
 	}
+
```
Comment (Kshitij-Katiyar) [inline server/webhook_worker.go:42]: these extra line this was some review comment or not ?

File: server/settings.go [hunk 3]
```diff
@@ -69,5 +70,5 @@ func (p *Plugin) settingsNotifications(header *model.CommandArgs, instanceID, ma
 		notifications = settingOn
 	}
 
-	return p.responsef(header, "Settings updated.\n\t%s notifications %s.", strings.Title(args[1]), notifications)
+	return p.responsef(header, "Settings updated.\n\t%s notifications %s.", cases.Title(language.Und, cases.NoLower).String(args[1]), notifications)
```
Comment (Kshitij-Katiyar) [inline server/settings.go:73]: need for cases and language import here? find something where we don't require 2 different packages for this statement

Comment (Nityanand13) [reply server/settings.go:73]: This is only how we can do.

**Review by Kshitij-Katiyar — APPROVED** [review]

File: server/settings.go (hunk 3 above)
Comment (manojmalik20) [reply server/settings.go:73]: What is the purpose of doing this?

Comment (Nityanand13) [reply server/settings.go:73]: In `Go 1.18` `strings.Title(str)` was deprecated, should change to `cases.Title(language.Und, cases.NoLower).String(str)`. You can look at here in the documenttion [strings.Title](https://pkg.go.dev/strings#Title)](https://pkg.go.dev/strings#Title)

**Review by manojmalik20 — APPROVED** [review]

**Pull request merged**